client/app/routes/home.tsx

```
@@ -8,6 +13,38 @@ export function meta({}: Route.MetaArgs) {
+export default function Home({ loaderData }: Route.ComponentProps) {
+  const postData = loaderData?.postData || ([] as PostData[]);
+  const [posts, setPosts] = useState(postData)
+  
+  return (
+    <div>
+      <SearchPosts setPosts={setPosts}/>
+      <h2>Posts</h2>
+      {posts.length > 0 ? (
+        <div className="posts-list flex flex-col gap-5">
+          {posts.map((post: PostData) => (
+            <Post post={post} />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)